YarnSpinnerConsole/ProgramExporter.cs

```diff
@@ -1,14 +1,11 @@
 ﻿using System;
-
+using System.Globalization;
 using Newtonsoft.Json;
 
 namespace Yarn
 {
-    public class ProgramExporter
+    public static class ProgramExporter
     {
-        public ProgramExporter()
-        {
-        }
```

Comment:
This class is only ever used statically, so I removed the unused constructor too. There's a warning about this now.